pytextnow/TNAPI.py

```diff
@@ -1,378 +1,376 @@
-        self.get_new_sms(self)
-        self.get_new_mms(self)
-
-    def get_new_sms(self, from_=None):
-        """
-        NOTE: This is ONLY called when the event listener hears something
-        TODO: Allow getting new messages from a specific person
-        """
-        newest_id = self.__db_handler._execute_sql(
-            "SELECT max (db_id) FROM '%s'" % (
-                self.__db_handler.get_table_name("Message")
-            )
-        ).first().id
-        # Query for all messages starting from the newest id we
-        # have on record
-        req = self.session.get(
-                "https://www.textnow.com/api/users/"
-                + self.__user.username
-                + "/messages?contact_value=%s"
-                + "&start_message_id=%s"
-                + "&direction=past"
-                + "&page_size=%s"
-                + "&get_archived=1" % (
-                    from_.number, newest_id, self.page_size
-                ),
-                headers=self.headers, cookies=self.cookies)
-        return ApiHandler.process_response(req)
-    
-    def await_result(self, func_name, fail_silently=False):
-        result = self.__api_handler.results.get(func_name, None)
-        if not result:
-            # Threading could have changed the value. check again
-            result = self.__api_handler.results.get(func_name, None)
-            # Ensure we don't go passed the timeout
-            start = time.time()
-            while not result:
-                result = self._api_handler.results.get(func_name, None)
-                if result:
-                    return result
-                if time.time() - start >= self.__await_result_timeout:
-                    if not fail_silently:
-                        raise Exception(
-                            "\n\nCellPhone timed out while waiting for the ApiHandler to "
-                            "finish executing '%s'.\n\n!!!Make sure your internet is connected!!!\n"
-                            "If your internet is slow, you may have to raise the timeout limit!" 
-                        )
-                    else:
-                        print(
-                            "\n\n!!!WARNING!!! CellPhone timed out while "
-                            + "awaiting the result of %s" % (func_name)
-                        )
+from playsound import playsound
+import atexit
+import datetime as dt
+import json
+import mimetypes
+import time
+from datetime import datetime, time
+
+import requests
+
+from pytextnow.TN_objects.contact import Contact
+from pytextnow.TN_objects.container import Container
+from pytextnow.TN_objects.error import FailedRequest, InvalidEvent
+from pytextnow.TN_objects.message import Message
+from pytextnow.TN_objects.multi_media_message import MultiMediaMessage
+from pytextnow.database.db import DatabaseHandler
+from pytextnow.tools.constants import *
+from pytextnow.tools.robot import RoboBoi
+
+class CellPhone:
```

Comment:
Could you tell me what all you changed in this file? Was it just the import or..?

Reply:
It was the imports and removing the hardcoded SQLite database path. It now just uses None as the default. `database/db.py` will check for None and replace it with the real default path.